src/interpolator.cpp

```diff
@@ -21,6 +21,9 @@ void Interpolator::init(const std::vector<double>& x_,
 // Compute the interpolated function from the points contained in 'x'
 // and 'y'.
 void Interpolator::initialise() {
+  if (not std::is_sorted(x.begin(), x.end())) {
+    util::stop("spline control points must be in non-descending order");
```

Comment:
non-descending -> ascending?

Also need to be unique? 

"spline control points must be unique and in ascending order"

Reply:
Yeah that's correct, the x points are required to be strictly ascending and therefore unique.

https://github.com/traitecoevo/plant/blob/a8f8a7b26059a7e82718bda5e85b5cfaa518b9ec/src/tk_spline.cpp#L151